### ISSUE TITLE
Peasy can't find p.exe on Windows

### ISSUE THREAD

**Issue by Vanlightly** [p-org/peasy-ide-vscode#75]
Peasy complains that P isn't installed so I have to use the VS Code terminal to run `p compile`. I looked at the Peasy code and located the problem.

The function `checkPInstalled` checks if the p binary exists but on Windows, it is an exe file and for some reason, the basename of the file is `p.exe` not `p`. 

So a fix will need to make sure it can detect the p.exe on Windows.

**Comment by Vanlightly** [comment]
Something like the following might do it:

```
var dirFiles = fs.readdirSync(dirPath);
    dirFiles.forEach(file => { 
      if (path.parse(file).name == "p") {
        return true;
      }
    })
```

**Comment by ankushdesai** [comment]
Thanks a lot Jack. We will fix this asap. thanks!

**Comment by mchadalavada** [comment]
Fix included in peasy-1.0.2
